conda_pack/core.py

```diff
@@ -813,8 +813,7 @@ def rewrite_shebang(data, target, prefix):
             new_shebang = '#!/usr/bin/env %s%s' % (executable_name,
                                                    options.decode('utf-8'))
             data = data.replace(shebang, new_shebang.encode('utf-8'))
-
-        return data, True
+            return data, True
```

Comment:
By moving this return inside the if block, we only return `True` if the shebang is _actually_ changed. There are several scripts that use simple `/bin/sh` shebangs but contain the prefix; those still need to be edited, but the current code fails to do so.